Running consent store in client mode triggers a nillptr dereference

```
$ NUTS_CSTORE_MODE=client NUTS_AUTH_MODE=client ./nuts-go registry register-vendor
...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x46a1241]
```

--- a/pkg/consent.go
+++ b/pkg/consent.go
@@ -130,7 +130,6 @@ func (cs *ConsentStore) Configure() error {
 				return
 			}
 		}
-
 	})
 
 	return err
@@ -138,18 +137,23 @@ func (cs *ConsentStore) Configure() error {
 
 //Shutdown closes the db connections
 func (cs *ConsentStore) Shutdown() error {
-	return cs.Db.Close()
+	if cs.Db != nil {
+		return cs.Db.Close()
+	}
+	return nil
 }
 
 // Start opens the db connections
 func (cs *ConsentStore) Start() error {
 	var err error
 
-	// gorm db connection
-	cs.Db, err = gorm.Open("sqlite3", cs.sqlDb)
+	if cs.Config.Mode == "server" {
+		// gorm db connection
+		cs.Db, err = gorm.Open("sqlite3", cs.sqlDb)
 
-	// logging
-	cs.Db.SetLogger(logrus.StandardLogger())
+		// logging
+		cs.Db.SetLogger(logrus.StandardLogger())
+	}
 
 	return err
 }

--- a/pkg/consent_test.go
+++ b/pkg/consent_test.go
@@ -624,7 +624,22 @@ func TestConsentStore_Configure(t *testing.T) {
 			t.Errorf("Expected error [%s], got [%v]", expected, err.Error())
 		}
 	})
+
+	t.Run("initialize store in client mode", func(t *testing.T) {
+		client := ConsentStore{
+			Config: ConsentStoreConfig{
+				Mode:             "client",
+			},
+		}
+
+		err := client.Configure()
+		assert.NoError(t, err)
+		err = client.Start()
+		assert.NoError(t, err)
+	})
 }
+
+
 
 func defaultConsentStore() ConsentStore {
 	client := ConsentStore{

